generators/CoreCLRStatefulActor/index.js

```diff
@@ -85,8 +85,8 @@ var ClassGenerator = generators.Base.extend({
  
     var appPackagePath = this.isAddNewService == false ? path.join(this.props.projName, appPackage) :  appPackage;
     var serviceSrcPath = this.isAddNewService == false ? path.join(this.props.projName, serviceProjName) : path.join(this.props.projName, serviceProjName) ;
-    var interfaceSrcPath = this.isAddNewService == false ? path.join(this.props.projName, interfaceProjName) : interfaceProjName;
-    var testClientSrcPath = this.isAddNewService == false ? path.join(this.props.projName, testClientProjName) : testClientProjName;
+    var interfaceSrcPath = this.isAddNewService == false ? path.join(this.props.projName, interfaceProjName) : path.join(this.props.projName, interfaceProjName);
```

Comment:
Am I missing something here? Both the cases looks same here.. Do we need the ternary operator?

Reply:
We dont need it. I made ad-hoc changes and forgot to remove them. Now removed.